track/orders.py

```diff
@@ -88,7 +88,8 @@ def _update_orders_with_shipping_info(tracking_info):
                 continue
             
             # if order has not shipped yet, we don't need to continue
-            if not tracking_info[order.order_number]['date_kit_shipped']: 
+            if not tracking_info[order.order_number]['date_kit_shipped']:
+                logger.error(f'Order {order.order_number} has no shipped date.') 
```

Comment:
I think we want 'warn' here. and in the three other log statements below as well.